src/github.com/getlantern/balancer/balancer.go

```diff
@@ -49,7 +68,25 @@ func New(dialers ...*Dialer) *Balancer {
 // remaining Dialers until it either manages to connect, or runs out of dialers
 // in which case it returns an error.
 func (b *Balancer) DialQOS(network, addr string, targetQOS int) (net.Conn, error) {
-	dialers := b.dialers
+	var dialers []*dialer
+
+	// Checking destination port.
+	_, port, _ := net.SplitHostPort(addr)
+
+	// Are we attempting to connect to port 80 (plain HTTP)?
```

Comment:
We actually support a few ports besides 80 and 443.  I don't know which of those are encrypted, or whether they pose any threat at all, but I thought I'd point it out:

https://github.com/getlantern/lantern/blob/9363758350d904e61753bc0e99a9f1edc49ec2b2/src/github.com/getlantern/flashlight/flashlight.go#L269